index.html

```diff
@@ -1369,25 +1369,20 @@ <h4>
     </p>
     <pre class="example">
       const reader = new NDEFReader();
-      await reader.scan({ recordType: "example.com:sp" });
+      await reader.scan({ recordType: "example.com:smart-poster" });
```

Comment:
Just as a note. Here we also have the choice to say `":smart-poster"`. According to the spec, `example.com` needs to match the the effective origin (though the algorithm is not there yet). If the app omits it, then the implementation will add it.

It would be important we get this right, so this policy needs discussion. If we want this, we need to add the origin-matching algorithm.

Reply:
Thinking about it again, I don't think Web NFC should be smart and use the effective domain of the origin as domain name in external type names when creating/reading external type records. 

I would argue web developers should always set the full external type record. Think about staging environments, where `staging.example.com:smart-poster` would be written automatically if `:smart-poster` were to be allowed. It would not match when read in a prod environment.
Does this make sense?

Reply:
I dont like the smartness either...

Reply:
Yes, the spec would be simpler that way.
So developers could give any 
a) free string we don't validate, only for the last part `:` + `<type-name>`
b) string that we validate to a domain name (we need an algorithm for it, not trivial)
c) a string that needs to validate to domain name that is part of the effective domain (algorithm needed).

Reply:
Rely on URL spec?

Reply:
![image](https://user-images.githubusercontent.com/1101339/70786496-13eb3f80-1d8d-11ea-8318-fc4bad52a1d1.png)

Reply:
So that means b + a. Fine for me.

Reply:
@zolkis Can you file an issue to track this?